pkgs/development/python-modules/git-dummy/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  installShellFiles,
+  setuptools,
+  gitpython,
+  typer,
+  pydantic-settings,
+}:
+
+buildPythonPackage rec {
+  pname = "git-dummy";
+  version = "0.1.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "initialcommit-com";
+    repo = "git-dummy";
+    rev = "v${version}";
+    hash = "sha256-viybxn2J7SO7NgSvjwlP+tgtm+H6QrACafIy82d9XEk=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    gitpython
+    typer
+    pydantic-settings
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd git-dummy \
+      --bash <($out/bin/git-dummy --show-completion bash) \
+      --fish <($out/bin/git-dummy --show-completion fish) \
+      --zsh <($out/bin/git-dummy --show-completion zsh)
+  '';
+
```

Comment:
Could you enable tests?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
The tests are currently broken, but I have added the parts to be able to run the tests.